sklearn/neighbors/_lof.py

```diff
@@ -150,6 +150,22 @@ class LocalOutlierFactor(KNeighborsMixin, UnsupervisedMixin,
         case, the offset is defined in such a way we obtain the expected
         number of outliers in training.
 
+    effective_metric_ : str
+        The effective metric used for the distance computation.
+        If `metric` is 'minkowski', `effective_metric_` is equivalent to
```

Comment:
I am not sure you want all these details. It is quite hard to keep in sync when the code changes. Maybe this is enough (not really sure)?
```
Effective metric used for the distance computation
```

Reply:
I agree with Loic. The details should be given in the Parameters. Once it has been fitted, we just want to know that this the metric used.